src/resources/js/flexible-tickets/modules/filters.js

```diff
@@ -216,3 +216,28 @@ wp.hooks.addFilter (
     'tec.tickets.flexibleTickets',
     filterTicketsControlsMappedProps
 )
+
+/**
+ *
+ * Forces the Prompt Message component to show on Recurring Events if they are part of a Series.
+ *
+ * @since TBD
+ *
+ * @param {ReactNode} component
```

Comment:
This is not a React node, this is the object representing the `mappedProp` of the Tickets Container component.

Reply:
This file was deleted, nothing to do here